lib/vagrant-openshift/command/test_origin_image.rb

```diff
@@ -117,14 +118,14 @@ def execute
 cd $temp_dir
 
 # clone the image repo
-git clone #{source}
-cd #{source_dir}
+git clone "#{source}"
```

Comment:
> is this the upstream (openshift) repo?

Oh well, on first thought I was thinking "origin", sorry.

The answer is **yes**, that's the upstream repo. The pull request ref is in the same repo.